src/test/java/edu/byu/cs/autograder/score/ScorerTest.java

```diff
@@ -201,7 +199,9 @@ void score__phaseNotGradeable() {
                 mockObserver, false);
         addQueueItem(new QueueItem("testNetId", Phase.Phase0, Instant.now(), true));
 
-        Rubric emptyRubric = new Rubric(null, null, null, true, "testNotes");
+        Rubric emptyRubric = new Rubric(new EnumMap<>(Rubric.RubricType.class),
```

Comment:
I've seen this constructor called multiple times, and the TODO commend and empty parameters seem duplicated to me. How would you feel about introducing a specialized static constructor instead?

I.E.
```java
// rubric.java
  public static Rubric withRubricType(Class<Rubric.RubricType> rubricType, boolean passed, string notes) {
    return new Rubric(new EnumMap<>(rubricType),
      null, null, null, //TODO: Remove these between semesters
      passed, notes);
  }

// Other files, like this one
Rubric emptyRubric = Rubric.withRubricType(Rubric.RubricType.class, true, "testNotes");
```

Reply:
I don't think it even needs to be a static factory method. Can just be a constructor matching what the normal constructor will look like after spring. Good idea, thank you.